data/korath/korath missions.txt

```diff
@@ -257,3 +257,332 @@ mission "Small Scale Delivery"
+			choice
+				`	(Drink it.)`
+					goto drink
+				`	(Make a motion as if drinking.)`
+			`	Either the Efret don't notice your ruse, or they are too polite to say. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
+				goto end
+			label drink
+			`	The drink is fermented and has a sweet and sour taste that leaves your lips and mouth tingling and then numb. They press the sweets on you so you take them. You see a few groups from the spaceport follow the same ritual, though some of them walk up to the "choir" and say something to them. When you finish the food, you take your leave and return to your ship.`
+			label end
+	on visit
+		dialog "You arrive, but you notice the <EDC-ship> is absent. You'll have to bring it here to finish the salvage mission."
+	npc save accompany
+		# Ship name should match the <EDC-ship> substitution above.
+		government "Kor Efret"
+		personality timid escort
+		ship "Charm-Shallop (Distressed)" "Esketari"
+
+mission "Efreti: Distress Call 8"
+	invisible
+	landing
+	source
+		attributes quarg
+	to offer
+		has "Efreti: Distress Call 7: done"
+	on offer
+		conversation
+			`The Quarg have translated the Korath language for you in the past. You have a data chip from when you helped the Efret salvage the <EDC-ship>. Would you like to see if the Quarg will tell you what the data chip says?`
+			choice
+				`	(Ask the Quarg to translate the data chip.)`
+				`	(Maybe another time.)`
+					defer
+				`	(I don't care what it says.)`
+					decline
+			
+			`	You approach one of the Quarg in the spaceport. It stops and acknowledges you but doesn't speak. It pauses for a few moments after you make your request but then holds out its hand. It takes the data chip and slots it into a communicator of some sort. It then nods to itself and hands you back the chip.`
+			# The message is translated from Efreti. Korath languages have no tenses due to their biology. It was translated by Quarg, who use archaic writing and wording. Hence the odd writing style.
+			`	You plug the chip into your own device and see two files: one in the Korath language, one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
```

Comment:
```suggestion
			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
```
The previous time you attempted to read the chip, you slotted it into your ship. Adjusted wording for that, instead of a magical device you happened to have had on you.